frontend/institution/registered_institution.html

```diff
@@ -2,9 +2,16 @@
     <div class="registered-institution-cover">
         <div ng-if="regInstCtrl.hasCoverPhoto()"
             style="background-image: url({{regInstCtrl.institution.cover_photo}});" id="card-cover-picture">
+            <div class="new-institution-tag-with-cover" md-colors="{background: 'teal-500'}" ng-if="regInstCtrl.hasNotSeenInstitution()">
+                <span><b>NOVO CADASTRO</b></span>
+            </div>
         </div>
-        <div ng-if="!regInstCtrl.hasCoverPhoto()" md-colors="{background: 'light-green'}">
+        <div ng-if="!regInstCtrl.hasCoverPhoto()" md-colors="{background: 'light-green'}" 
+            id="default-reg-inst-cover">
             <img ng-src="{{'app/images/institution.png'}}">
+            <div class="new-institution-tag" md-colors="{background: 'teal-500'}" ng-if="regInstCtrl.hasNotSeenInstitution()">
```

Comment:
why its duplicated?

Reply:
This tag, "novo cadastro", appears in the institution's cover, in this html there are two options for the cover. The first one is the institution's cover if there is one, otherwise the default cover is applied. That's the reason I need to duplicate the tag. Besides, the css is diferent once the default cover is a <img>  and the first option is a background-image set in the style